python/ray/_private/node.py

```diff
@@ -317,6 +324,41 @@ def __init__(
         self.validate_ip_port(self.gcs_address)
         self._record_stats()
 
+    def check_persisted_session_name(self):
```

Comment:
I'd prefer moving it to the cython file. But I think it's fine if we'll change this part after the release blockers.